sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -233,6 +233,10 @@ export class ApplicationStudentsController extends BaseController {
       payload.data.selectedOfferingEndDate = studyEndDate;
     }
 
+    // Set studentDataAdditionalTransportKm to 0 for Camunda assessment calculation
+    // if it is not provided from the student application payload
+    payload.data.studentDataAdditionalTransportKm =
```

Comment:
As discussed please handle the null checks at the workflow level.

Reply:
The dynamic data is removed here and added in the workflow level. Please have a look.